coff.ml

```diff
@@ -1038,31 +1038,34 @@ end
 
 
 module Stacksize = struct
-  let set_stack_reserve filename reserve =
+  let set_stack_reserve filename size_of_stack_reserve =
     let filename =
       if not (Sys.file_exists filename) && (Sys.file_exists (filename ^ ".exe")) then filename ^ ".exe"
       else filename
     in
     let ic = open_in_bin filename in
-    let hdr_offset = int16 (read ic 0x3c 2) 0 in
-    let pe_signature = read_str ic hdr_offset 4 in
+    let pe_signature_offset = int32 (read ic 0x3c 4) 0 |> Int32.to_int in
```

Comment:
If the offset fits in 16 bits, I suppose it's safe to forget the high-order bit from int32 to int.